gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -211,6 +211,8 @@ private static void configureErrorProneOptions(
             // https://github.com/google/error-prone/issues/1106
             errorProneOptions.check("TypeParameterUnusedInFormals", CheckSeverity.OFF);
             errorProneOptions.check("PreferCollectionConstructors", CheckSeverity.OFF);
+            errorProneOptions.check("UnnecessaryLambda", CheckSeverity.OFF);
+            errorProneOptions.check("ThrowSpecificity", CheckSeverity.OFF);
```

Comment:
These are the two where we've seen failures on java13 projects so far, but all checks using SuggestedFixes.qualifyType and SuggestedFixes.prettyType are impacted:

```
[gradle-baseline]$ git grep "SuggestedFixes\..*Type"
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/AssertjPrimitiveComparison.java:        String cast = '(' + SuggestedFixes.prettyType(state, null, targetType) + ") ";
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/CatchSpecificity.java:                                .map(type -> SuggestedFixes.prettyType(state, fix, type))
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/LambdaMethodReference.java:        return toMethodReference(SuggestedFixes.qualifyType(state, builder, symbol))
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/MoreASTHelpers.java:                .sorted(Comparator.comparing(type -> SuggestedFixes.prettyType(state, null, type)))
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferAssertj.java:                            + SuggestedFixes.qualifyType(state, fix, "org.assertj.core.api.HamcrestCondition")
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferAssertj.java:                            + SuggestedFixes.qualifyType(state, fix, "org.assertj.core.api.HamcrestCondition")
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferAssertj.java:            String qualifiedMap = SuggestedFixes.prettyType(
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferAssertj.java:            return SuggestedFixes.qualifyType(state, fix, "org.assertj.core.api.Assertions.assertThat");
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferBuiltInConcurrentKeySet.java:            String qualifiedType = SuggestedFixes.qualifyType(state, fix, "java.util.concurrent.ConcurrentHashMap");
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferCollectionConstructors.java:        String collectionType = SuggestedFixes.qualifyType(state, fixBuilder, collectionClass.getName());
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferCollectionTransform.java:                    qualifiedType = SuggestedFixes.qualifyType(state, fix, "com.google.common.collect.Lists");
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferCollectionTransform.java:                    qualifiedType = SuggestedFixes.qualifyType(state, fix, "com.google.common.collect.Collections2");
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferListsPartition.java:                String qualifiedType = SuggestedFixes.qualifyType(state, fix, "com.google.common.collect.Lists");
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferSafeLoggingPreconditions.java:        String logSafeQualifiedClassName = SuggestedFixes.qualifyType(state, fix, "com.palantir.logsafe.Preconditions");
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/ReadReturnValueIgnored.java:        String qualifiedReference = SuggestedFixes.qualifyType(state, fix, fullyQualifiedReplacement);
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/StreamOfEmpty.java:                    .addFix(MoreSuggestedFixes.renameInvocationRetainingTypeArguments(tree, "empty", state))
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/StrictCollectionIncompatibleType.java:        return SuggestedFixes.prettyType(null, null, type);
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/ThrowError.java:            String qualifiedName = SuggestedFixes.qualifyType(state, fix, IllegalStateException.class.getName());
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/ThrowError.java:            String qualifiedName = SuggestedFixes.qualifyType(state, fix,
baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/ThrowSpecificity.java:                        .map(checkedException -> SuggestedFixes.prettyType(state, fix, checkedException))
```

Should we disable all these checks on java13? We should probably migrate our calls to our `MoreSuggestedFixes` utility with a fallback for java13 which simply adds an import and returns the class name. It would be nice to get tests running on java13 in this repo so we ca be more confident moving forward.

Reply:
Yeah that seems much more reasonable - too much late night hacking for Dan.

Reply:
Done here: https://github.com/palantir/gradle-baseline/pull/1110

I think we still may want to disable UnnecessaryLambda, UnnecessaryAbstractClass, etc on jdk13 because we can't modify upstream code.